CTP/README.md

```diff
@@ -97,139 +114,163 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
-	* Controller node configuration:
-	
-		```
-		One or more HA test environments are configured in CTP/conf/ha_repl.conf. 
-		There are two HA test environments instance01 and instance02 as below:
-		env.instance01.master.ssh.host=<master ip>
-		env.instance01.master.ssh.user=<ssh user>
-		env.instance01.slave1.ssh.host=<slave ip>
-		env.instance01.slave1.ssh.user=<ssh user>
-		env.instance01.cubrid.cubrid_port_id=<cubrid port>
-		env.instance01.ha.ha_port_id=<ha port>
-		env.instance01.broker.BROKER_PORT=<broker port>
-	
-		env.instance02.master.ssh.host=<master ip>
-		env.instance02.master.ssh.user=<ssh user>
-		env.instance02.slave1.ssh.host=<slave ip>
-		env.instance02.slave1.ssh.user=<ssh user>
-		env.instance02.cubrid.cubrid_port_id=<cubrid port>
-		env.instance02.ha.ha_port_id=<ha port>
-		env.instance02.broker.BROKER_PORT=<broker port>
-		```
-		
-		```
-		Test cases are placed in controller node and configured like below:
-		main.testcase.root=/path/to/testcases/
-		```
-			
-		```
-		The test build should be provided with URL link:
-		main.testbuild.url=http://127.0.0.1/REPO_ROOT/store_01/10.1.0.6929-b049ba5/drop/CUBRID-10.1.0.6929-b049ba5-Linux.x86_64.sh	
-		```
-		
-		Regarding more parameters for ``HA Replication`` test, please refer to [CTP/conf/ha_repl.conf](conf/ha_repl.conf)	
-			
-	* HA test environment configuration (master and slave node):
+	* Prepare environments (e.g., one account ``controller`` as controller, one account ``ha_repl_instance1`` as a master test instance, one same account name ``ha_repl_instance1`` on the other machines as a slave test instance)
+	* Check out CTP for each account and configure environment variables for CTP ``controller`` and test instances ``ha_repl_instance1`` (both master and slave)
+      ```
+	  // on controller node
+	  JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
+
+	  // on test instance nodes (master and slave)
+	  CTP_HOME (e.g., export CTP_HOME=$HOME/CTP)
+	  JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
+      ```	
+	* **Example** ``ha_repl.conf`` for test instances (master and slave), scenario and test build:
+      ```
+	  # Test instance information:
+	  env.instance1.master.ssh.host=192.168.1.10
+	  env.instance1.master.ssh.user=ha_repl_instance1
+	  env.instance1.slave1.ssh.host=192.168.1.11
+	  env.instance1.slave1.ssh.user=ha_repl_instance1
+	  env.instance1.cubrid.cubrid_port_id=1137
+	  env.instance1.ha.ha_port_id=59001
+	  env.instance1.broker.BROKER_PORT=35000
+      ```	
 	  
+      ```
+	  # Define the path of test cases used for testing, it should be checked out on controller node
+	  main.testcase.root=/home/controller/testcases/sql
+      ```
+	NOTE: the scenario must be checked out on ``controller`` account, not on the test instance
+
+      ```
+	  # Define the URL of test build which will be used to test
+	  # If set 'main.deploy.rebuild_yn' as 'false', this parameter will be ignored

```

Comment:
"If set 'main.deploy.rebuild_yn' as 'false', this parameter will be ignored"

Please correct it.